venv/Hometask_1.py

```diff
@@ -0,0 +1,38 @@
+
+import random # importing random module
+# getting 100 numbers from 0 to 1000
+list = [] # create a list
+for i in range(100):  # define count of numbers in the list
+    list.append(random.randint(0, 1001))  # create the list of numbers from 0 to 1000
+# sort list from min to max
+new_list = [] # create a new list for further sorted numbers from min to max
+while list:
+    minimum = list[0]  # define min value in the list
```

Comment:
'define min value in the list' isn't quite correct. Because you put first value from the list (list[0]) to this variable and not the minimum. And after that you start to compare numbers with this value.